# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.h
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae editor/libeditor/HTMLEditor.h
--- a/editor/libeditor/HTMLEditor.h	Tue Jun 17 00:10:27 2025 +0000
+++ b/editor/libeditor/HTMLEditor.h	Tue Jun 17 00:37:22 2025 +0000
@@ -1068,9 +1068,6 @@
       return mNewInvisibleWhiteSpaces;
     }
 
-    void MaybeAppendNewInvisibleWhiteSpace(
-        const nsIContent* aContentWillBeRemoved);
-
    private:
     ~DocumentModifiedEvent() = default;
 